src/controllers/reviews/getreviews.ts

```diff
@@ -0,0 +1,15 @@
+import { NextFunction, Request, Response } from 'express';
+import Review from '../../database/reviews';
+import CustomError from '../../helpers';
+
+const getReviews = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+  try {
+    const productID = req.params.id;
+    const productReviews = await Review.find({ productID });
+    res.status(200).json(productReviews);
```

Comment:
send data as an object with a message that getting all reviews about 1 product successfully